gcc-c14.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2013-2017, Ruslan Baratov
+# All rights reserved.
+
+if(DEFINED POLLY_GCC_C14_CMAKE_)
+  return()
+else()
+  set(POLLY_GCC_C14_CMAKE_ 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_init.cmake")
+
+polly_init(
+    "gcc / c++14 support / C11"
+    "Unix Makefiles"
+)
+
+include("${CMAKE_CURRENT_LIST_DIR}/utilities/polly_common.cmake")
+
+include("${CMAKE_CURRENT_LIST_DIR}/compiler/gcc.cmake")
+include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx14.cmake")
```

Comment:
Name of the toolchain doesn't reflect this, probably it's better should be `gcc-cxx14-c11`. `cxx11` part is missing in `gcc-c11` because it's kind of a default for now.

Reply:
Updated with requested change.